source/Releases/Release-Foxy-Fitzroy.rst

```diff
@@ -288,6 +288,36 @@ There are copies in `example_interfaces <https://index.ros.org/p/example_interfa
 - ``std_msgs/msg/UInt8``
 - ``std_msgs/msg/UInt8MultiArray``
 
+sros2
```

Comment:
This was a fairly wide-ranging change that wasn't just sros2. Can we just say:

```suggestion
Security features
```

instead?

Reply:
I was wondering the same thing. Most sections seem to be package-specific but there are some non package-specific sections as well, so let's rename the section and see what the reviewers say